### PR TITLE
fix: raise an error when the asname of import statment is defined multi-times

### PR DESCRIPTION
**Pull request by zong-zhe**
#### 1. Does this PR affect any open issues?(Y/N) and add issue references (e.g. "fix #123", "re #123".):

- [ ] N
- [x] Y 

fix issue #441 

#### 2. What is the scope of this PR (e.g. component or file name):

kclvm/sema/src/resolver/import.rs

#### 3. Provide a description of the PR(e.g. more details, effects, motivations or doc link):

- [x] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Other

#### 4. Are there any breaking changes?(Y/N) and describe the breaking changes(e.g. more details, motivations or doc link):

- [x] N
- [ ] Y 

#### 5. Are there test cases for these changes?(Y/N) select and add more details, references or doc links:

- [x] Unit test
- [ ] Integration test
- [ ] Benchmark (add benchmark stats below)
- [ ] Manual test (add detailed scripts or steps below)
- [ ] Other

### CODE REVIEW

File: kclvm/cmd/src/tests.rs [hunk 1]
```diff
@@ -596,3 +596,22 @@ fn test_error_message_fuzz_unmatched() {
         }
     }
 }
+
+#[test]
+fn test_multi_import() {
```
Comment (Peefy) [inline kclvm/cmd/src/tests.rs:601]: I still suggest placing this test in the resolver. The cmd test is a bit too much.

Comment (zong-zhe) [reply kclvm/cmd/src/tests.rs]: fixed.

**Review by Peefy — APPROVED** [review]
LGTM

File: kclvm/sema/src/resolver/import.rs [hunk 2]
```diff
@@ -91,6 +91,19 @@ impl<'ctx> Resolver<'ctx> {
                             {
                                 match self.ctx.import_names.get_mut(&self.ctx.filename) {
                                     Some(mapping) => {
+                                        // 'import sub as s' and 'import sub.sub as s' will raise this error.
+                                        // 'import sub' and 'import sub' will not raise this error.
+                                        if mapping.get(&import_stmt.name).is_some()
```
Comment (Peefy) [inline kclvm/sema/src/resolver/import.rs:96]: The PR did not check if it imported the same package, which is not an error.

**Pull request merged**